build-system/tasks/ava.js

```diff
@@ -26,6 +26,7 @@ async function ava() {
   // (see targetMatchers[Targets.AVA])
   const testFiles = [
     'build-system/tasks/get-zindex/get-zindex.test.js',
+    'build-system/tasks/make-extension/test/test.js',
```

Comment:
@rsimha Do I need to list this file somewhere to setup CI targets?

Reply:
Yes. Follow the instructions in the comment above (lines 25-26) and add a case to `build-targets.js`.

Reply:
I don't know who that is. (thanks!)